source/reference/minio-mc-admin/mc-admin-user.rst

```diff
@@ -13,12 +13,11 @@
 Description
 -----------
 
-.. start-mc-admin-user-desc
+.. start-mc-admin-user-desc                                                                    
 
-The :mc:`mc admin user` command manages users on a MinIO deployment. Clients
-*must* authenticate to the MinIO deployment with the access key and secret key
-associated to a user on the deployment. MinIO users constitute a key component in
-MinIO Identity and Access Management.
+The :mc:`mc admin user` command manages users on a MinIO deployment.
+Clients *must* authenticate to the MinIO deployment with the access key and secret key associated to a user on the deployment.
+MinIO users constitute a key component in MinIO Identity and Access Management.
 
 .. end-mc-admin-user-desc
 
```

Comment:
Might be worth mentioning that users can also authenticate through an external IDP using either OIDC or AD/LDAP (with links to those pages).

Reply:
Link to the respective `mc admin idp` pages.

Reply:
> Alternatively, you can set up an external Identity Provider using either :mc:`OIDC <mc admin idp openid>` or :mc:`Active Directory / LDAP <mc admin idp ldap>`.

Reply:
@djwfyi Edited to clarify other types of IDP users are managed by their respective commands. Not entirely sure I have the language correct, staged here:
http://192.241.195.202:9000/staging/DOCS-785/linux/html/reference/minio-mc-admin/mc-admin-user.html#description

Related: is this statement important to include here? I'm not convinced it adds practical information to the page, particularly with new references to external user auth.
> MinIO users constitute a key component in MinIO Identity and Access Management.

Reply:
Haha did not see your comment while I was writing mine 😄

Reply:
Additional edits to tweak this and related text.